jp.go

```diff
@@ -119,7 +126,12 @@ func runMain(c *cli.Context) int {
 		if c.Bool("compact") {
 			toJSON, err = json.Marshal(result)
 		} else {
-			toJSON, err = json.MarshalIndent(result, "", "  ")
+			if color.NoColor {
```

Comment:
What is the value of `color.NoColor` ?

Reply:
`color.NoColor` is  a boolean.

I had forgotten to initialize and document the value.

Reply:
But you're not supposed to write to the package var right ?